cfme/tests/configure/test_docs.py

```diff
@@ -85,7 +80,7 @@ def test_contents(appliance, soft_assert):
         href = view.browser.get_attribute(attr='href',
                                           locator=doc_widget.link.locator)
         data = requests.get(href, verify=False)
-        pdf_titlepage_text_low = pdf_get_text(StringIO(data.content), [0]).lower()
+        pdf_titlepage_text_low = pdf_get_text(BytesIO(data.content), [0]).lower()
```

Comment:
PRT failing here with:
```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
cfme/tests/configure/test_docs.py:37: in pdf_get_text
    interpreter.process_page(page)
/usr/lib/python2.7/site-packages/pdfminer/pdfinterp.py:833: in process_page
    self.device.end_page(page)
/usr/lib/python2.7/site-packages/pdfminer/converter.py:37: in end_page
    self.receive_layout(self.cur_item)
/usr/lib/python2.7/site-packages/pdfminer/converter.py:171: in receive_layout
    render(ltpage)
/usr/lib/python2.7/site-packages/pdfminer/converter.py:161: in render
    render(child)
/usr/lib/python2.7/site-packages/pdfminer/converter.py:161: in render
    render(child)
/usr/lib/python2.7/site-packages/pdfminer/converter.py:161: in render
    render(child)
/usr/lib/python2.7/site-packages/pdfminer/converter.py:163: in render
    self.write_text(item.get_text())
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <PDFDevice>, text = 'R'

    def write_text(self, text):
>       self.outfp.write(text.encode(self.codec, 'ignore'))
E       TypeError: unicode argument expected, got 'str'

/usr/lib/python2.7/site-packages/pdfminer/converter.py:154: TypeError
```

Perhaps this arg passed into `pdf_get_text` should remain a `StringIO` object?